features/step_definitions/tournament.rb

```diff
@@ -111,3 +115,25 @@ def placing_to_display_string(placing)
   visit event_path single_tournament
   expect(page).to have_text("#{placing_to_display_string placing} #{team.name}")
 end
+
+When(/^the schedule page is visited$/) do
+  visit event_schedule_path(single_tournament)
+end
+
+Given(/^(\d+) teams join the tournament$/) do |num_teams|
+  for each in 1..num_teams do
+    single_tournament.add_team create_team
+  end
+end
+
+And(/^he fills in valid tournament data$/) do
+  fill_in :tournament_name, with: 'Dummy'
+  fill_in :tournament_discipline, with: 'DummySport'
+  fill_in :tournament_max_teams, with: 8
+  page.select I18n.t('activerecord.attributes.event.player_types.team'), from: :event_player_type
```

Comment:
And you could look at the other selection tests in rspec for a cleaner way to select these. Maybe `:team` and `from: Event.human_attribute_name(...)`.